### PR TITLE
NGINX Configuration Module - closes #167

### LINKED ISSUE #167: NGINX Configuration Module

### PR DESCRIPTION
**Pull request by bob-crutchley**

### CODE REVIEW

File: topics/nginx/modules/configuration/README.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+The `include` directive can be used to effectively "import" other configurations from other files.
+You can use this for feature-specific configuration files; conventionally you would store these extra files in `/etc/nginx/conf.d` on Linux systems and `[INSTALL_FOLDER]/conf` on Windows.
+Here is an example for the `conf.d/http` and `conf.d/stream` files being included:
+```text
+include conf.d/http;
+include conf.d/stream;
+```
+
+## Top-Level Contexts
+There are a few top-level contexts that are used to group together directives for handling different traffic types:
```
Comment (tvaidotas) [inline topics/nginx/modules/configuration/README.md]: could you give example use cases in what scenarios would you use these traffic types

File: topics/nginx/modules/configuration/README.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+|Request|Response|
+|-------|--------|
+|http://localhost:8001/one|Location 1, Virtual Server 1|
+|http://localhost:8002/two|Location 2, Virtual Server 1|
+|http://localhost:9001/one|Location 1, Virtual Server 2|
+|http://localhost:9002/two|Location 2, Virtual Server 2|
+
+### Mail & Stream
+For the Mail and TCP/UDP traffic the `mail` and `stream` contexts are used respectively.
```
Comment (tvaidotas) [inline topics/nginx/modules/configuration/README.md]: what's a stream, in what scenario would you use it

Comment (bob-crutchley) [reply topics/nginx/modules/configuration/README.md]: added something short for this

File: topics/nginx/modules/configuration/README.md (hunk 1 above)
Comment (bob-crutchley) [reply topics/nginx/modules/configuration/README.md]: I've added a couple of simple scenarios to the bullet list to elaborate on these a little more

**Pull request merged**